src/control/lib/daos/container_property.go

```diff
@@ -549,7 +549,16 @@ var propHdlrs = propHdlrMap{
 	// ----------------------------------------
 	C.DAOS_PROP_ENTRY_LAYOUT_TYPE: {
 		C.DAOS_PROP_CO_LAYOUT_TYPE,
-		"Layout Type",
+		func() string {
+			acc := []string{}
+			for i := 0; i < C.DAOS_PROP_CO_LAYOUT_MAX; i++ {
+				var loStr [10]C.char
+
+				C.daos_unparse_ctype(C.ushort(i), &loStr[0])
+				acc = append(acc, C.GoString(&loStr[0]))
+			}
+			return "Layout Type (" + strings.Join(acc, ", ") + ")"
```

Comment:
it would be good to have a test for this logic but I wouldn't block on it

Reply:
> * [x]  Add unit tests

Fixed with commit 8879a2d62